recipes/flit-scm/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "flit-scm" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/{{ name[0] }}/{{ name }}/flit_scm-{{ version }}-py3-none-any.whl
```

Comment:
Source is not available for this release in any form other than a wheel (either from PyPI or the GitLab URL).  I have requested that a tag be made:
https://gitlab.com/WillDaSilva/flit_scm/-/issues/3